src/app/components/user-options/user-options.component.ts

```diff
@@ -75,4 +78,22 @@ export class UserOptionsComponent implements OnInit {
     }
     return this.password.length < 6;
   }
+
+  cancel(input) {
```

Comment:
what does this function do? what means this argument? what is happening here? please choose correct names.